roles/cifmw_cephadm/README.md

```diff
@@ -7,7 +7,7 @@ The
 [openstack-k8s-operators HCI documentation](https://github.com/openstack-k8s-operators/docs/blob/main/hci.md)
 describes how to run Ceph on EDPM nodes but leaves it to the reader
 to install Ceph with `cephadm`. The `cifmw_cephadm` role and
-`ceph.yml` playbook may be used to automate the Ceph installation.
+`hooks/playbooks/ceph.yml` playbook may be used to automate the Ceph installation.
```

Comment:
maybe s/playbook/hook/ ?

Reply:
why? This is the real path for that play...

Reply:
but... Still it is a playbook in hook location :D I can change to `hook playbook`. ok?

Reply:
Uhm, https://github.com/openstack-k8s-operators/ci-framework/blob/5703bdde587294121499365c3da7a82a063fdb90/hooks/playbooks/README.md generally talks about "hooks".

And I guess hooks/playbooks/README.md should be updated as well?